test/functional/rpc_fundrawtransaction.py

```diff
@@ -543,7 +543,7 @@ def test_locked_wallet(self):
         self.nodes[1].getnewaddress()
         self.nodes[1].getrawchangeaddress()
         inputs = []
-        outputs = {self.nodes[0].getnewaddress():1.09999500}
+        outputs = {self.nodes[0].getnewaddress():1.19999500}
```

Comment:
Note: I believe this change is related to #22686. What I think was happening in previous revisions of this PR is that due to that bug, `KnapsackSolver` would find a solution without change that doesn't actually work, and because of that, it would also have worse waste than the BnB solution. We don't see this in master because there is a BnB solution and in master, we default to the BnB solution. This PR, combined with #22686 allows `KnapsackSolver` to find a solution with change that happens to have less waste, so it was being chosen, causing the test to fail at the `fundrawtransaction` below. So this value is being changed to just under 1.2 which is where an optimal input set is being found.

Reply:
I recompiled the old version of this PR and confirmed that